### PR TITLE
Housekeeping and setting up workflow more completely

### PR DESCRIPTION
**Pull request by SomethingNew71**

### CODE REVIEW

File: .eslintrc.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
+module.exports = {
```
Comment (Sntax) [inline .eslintrc.js]: Why use `.eslintrc.js` instead of a regular `.eslintrc`? What are the benefits out of curiosity?

File: package.json [hunk 2]
```diff
@@ -1,55 +1,81 @@
 {
+  "name": "classicminidiy",
+  "description": "![alt text](http://i.imgur.com/E86umcV.png?3 \"CMDIY Logo\")",
```
Comment (Sntax) [inline package.json]: Inline markdown image in a `package.json` description?

File: src/app/main/main.scss [hunk 3]
```diff
@@ -28,6 +28,7 @@
   white: white;
```
Comment (Sntax) [inline src/app/main/main.scss]: is `white: white;` valid?

File: .eslintrc.js (hunk 1 above)
Comment (icdevin) [inline .eslintrc.js]: `.eslintrc` is deprecated.

Comment (Sntax) [inline .eslintrc.js]: Holy shit, looks like its been deprecated for at least a year too. How did I miss that? Good to know!

Comment (Sntax) [inline .eslintrc.js]: As an alternative you can add ` "eslintConfig": { ... }` to your `package.json`. TIL.

File: src/app/main/main.scss (hunk 3 above)
Comment (SomethingNew71) [inline src/app/main/main.scss]: Lol probably not..... Where did that even come from?  Silly app generators.

File: package.json (hunk 2 above)
Comment (SomethingNew71) [inline package.json]: `npm init` must have done this smh

**Pull request merged**